### ISSUE TITLE
Some Server Issues

### ISSUE THREAD

**Issue by Darkosto** [Vanhal/Zoooooom#1]
Hello! 

Firstly, thank you for providing an extremely easy to use and functional piping system for 1.8.9 :)

I am currently using Zoooooom-1.8.9-0.1.1

I've noticed some issues, mostly visual glitches, while playing on a server (linux based). When a new user joins the server, all zoooooom pipes will visably disconnect from each other as if they are individual pipe segments: https://sx.artdude.uk/hotlink-ok/darkosto/c9lD-000f7-1458247854.png but still function correctly. This is RF, Item, and liquid pipes. Leaving the server and reconnecting seems to fix the issue.

Also, in that same image, I had a water source block that was moved and caused all pipes to start visually "freaking out": https://secure.twitch.tv/darkosto/v/54999451?t=01h03m15s

As I mentioned, all issues seem to be visual only.

Thanks for looking into this :)
Darkosto

**Comment by Vanhal** [comment]
Thanks for the bug report, some super weird looking stuff happening there, I'll have a look into it over the next few days and try and reproduce it.

**Comment by Vanhal** [comment]
Think I may have to make some big changes to fix this properly. I've made some changes so that the unconnected pipes should update themselves after a while without needing to reconnect or anything.

**Comment by Darkosto** [comment]
Thank you for checking this out!! :)

Also, do you have plans to move to 1.9 at all? Just wondering :)

**Comment by Vanhal** [comment]
Yeah I gave it a shot earlier today but the forge mutli models don't seem to be ported over yet so I'm gonna have to wait a bit
